salt/edx/mitx_etl.sls

```diff
@@ -0,0 +1,57 @@
+install_python3_pip:
+  pkg.installed:
+    - pkgs:
+        - python3
```

Comment:
Since we're depending on Git to load the etl repo we should pass it as a required package here and maybe rename the id to something like `install_etl_os_dependencies`

Reply:
Added git pkg and renamed state id